packages/sdk/src/react/ui/components/_internals/action-button/ActionButton.tsx

```diff
@@ -34,15 +36,29 @@ export const ActionButton = observer(
 		tokenId,
 		action,
 		highestOffer,
+		lowestListing,
 	}: ActionButtonProps) => {
 		const { show: showCreateListingModal } = useCreateListingModal();
 		const { show: showMakeOfferModal } = useMakeOfferModal();
 		const { show: showSellModal } = useSellModal();
 		const { show: showTransferModal } = useTransferModal();
+		const { show: showBuyModal } = useBuyModal();
 
 		if (action === CollectibleCardAction.BUY) {
-			console.log('Buy action');
-			return;
+			if (!lowestListing)
```

Comment:
By doing this, does it mean we always expect users to checkout with lowest listing price?
If there are two listings: 10 usdc, 20 usdc, and user clicked buy on 20 usdc one, shouldn't we notify them before showing a modal with 10 usdc? May it become confusing?

Reply:
More of a product question, but this is how the rest of the UI works.
If you hit a buy or sell button that is not on the order/listing it will select the 'best' deal for you.
So highest offer is you want to sell and lowest listing if you want to buy

Reply:
Ah, fair point. Thanks for explanation 🙌🏼